File: TransCoder_tokenized_test_set_functions/cpp/OVERLAPPING_SUM_TWO_ARRAY.cpp
int findSum ( int A [ ], int B [ ], int n ) {
  unordered_map < int, int > hash;
  for ( int i = 0;
  i < n;
  i ++ ) {
    hash [ A [ i ] ] ++;
    hash [ B [ i ] ] ++;
  }
  int sum = 0;
  for ( auto x : hash ) if ( x . second == 1 ) sum += x . first;
  return sum;
}

File: TransCoder_tokenized_test_set_functions/python/OVERLAPPING_SUM_TWO_ARRAY.py
def findSum ( A, B, n ) :
    Hash = defaultdict ( lambda : 0 )
    for i in range ( 0, n ) :
        Hash [ A [ i ] ] += 1
        Hash [ B [ i ] ] += 1
    Sum = 0
    for x in Hash :
        if Hash [ x ] == 1 :
            Sum += x
    return Sum
